invisible_cities/calib/sipm_pdf.py

```diff
@@ -22,7 +22,7 @@
 from ..  cities.base_cities import EventLoop
 
 
-class SipmPdf(CalibratedCity):
+class Sipm_pdf(CalibratedCity):
```

Comment:
We conform to PEP8 when it comes to class names (unless there is some overriding reason not to), so this change would have to be reverted, were it not for the fact that it's not a Calvino name, so @jjgomezcadenas is not going to accept the old name either.